style.css

```diff
@@ -492,6 +492,318 @@ form {
+}
+
+.modal-content .project-description {
+  padding: 15px 0;
+}
+
+.project-information-2 {
+  width: 100%;
+}
+
+.modal-content .divider {
+  width: 100%;
+  border: 0;
+  border-bottom: 1px solid #ebecf0;
+  margin: 15px 0;
+}
+
+.modal-actions {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  width: 100%;
+}
+
+.modal-actions .project-btn {
+  width: 100%;
+  text-align: center;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+  padding: 14px;
+}
+
+.modal-actions a button:hover img {
+  filter: brightness(9.5);
+}
+
+.modal-actions a button {
+  display: flex;
+  align-items: center;
+  gap: 5px;
+}
+
+.project-additional-info {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  margin-bottom: 0;
+}
+
+.project-additional-info h5 {
+  margin: 0;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.project-additional-info p {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin: 0;
+}
+
+.details {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.popup-canopy {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 236px;
+  height: 32px;
+}
+
+.popup-img {
+  width: 311px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.popup-description {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  width: 311px;
+  height: 353px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.popup-text {
+  width: 311px;
+  height: 220px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.popup-tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 15px;
+  width: 53px;
+}
+
+.tags-item {
+  width: 73px;
+  height: 24px;
+  padding: 4px 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.popup-divider {
+  width: 311px;
+  height: 17px;
+}
+
+.popbtn {
+  width: 311px;
+  height: 56px;
+}
+
+.btn-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+  height: auto;
+}
+
+.see-live {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 22px;
+  gap: 10px;
+  position: absolute;
+  width: 136px;
+  height: 48px;
+  left: 9px;
+  bottom: 4px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
+
+.see-source {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 14px;
+  gap: 10px;
```

Comment:
-[ ] You did well displaying the pop-up with javascript as instructed. however, the source and live link buttons are hidden both on the desktop and mobile versions of the app 😉  .  You'd agree with me that this does not give a good user experience to any intending viewer, and might cause a disinterest lets to say to a hiring manager who wants to see the `source link` or `live version` of each app.  kindly readjust this on both mobile and desktop 👍 

# Your pop-up view on the desktop:
![image](https://user-images.githubusercontent.com/50990925/216263334-1f930f89-48e1-47ba-878d-b3a050a7aca0.png)

# Expected view based on figma design: 
- All features should be visible without a scroll :
![image](https://user-images.githubusercontent.com/50990925/216263663-23cedee0-5cfa-48e5-bb2b-90880758e7b0.png)

# You view mobile:
![image](https://user-images.githubusercontent.com/50990925/216263814-3fb1d48d-99e5-4b7f-a33f-cc9eceebed6d.png)

# Expected view:
![image](https://user-images.githubusercontent.com/50990925/216263945-d36a6430-c173-46dc-aebf-898e64d446cd.png)


- In case you fill such, you can check a [Sample project](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) Of how this was implemented by a fellow student